specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/DelegatedProviderOffer.json

```diff
@@ -0,0 +1,214 @@
+            "properties": {
+                "delegatedOfferId": {
+                    "description": "The delegated offer identifier.",
+                    "type": "string"
+                },
+                "displayName": {
+                    "description": "Display name of offer.",
+                    "type": "string"
+                },
+                "description": {
+                    "description": "Description of offer.",
+                    "type": "string"
+                },
+                "externalReferenceId": {
+                    "description": "External reference identifier.",
+                    "type": "string"
+                },
+                "accessibilityState": {
+                    "description": "Offer accessibility state.",
+                    "$ref": "#/definitions/AccessibilityState"
+                },
+                "subscriptionCount": {
+                    "description": "Current subscription count.",
+                    "type": "integer"
+                }
+            }
+        },
+        "AccessibilityState": {
+            "type": "string",
+            "description": "Represents the state of the offer",
+            "enum": [
+                "Private",
+                "Public",
+                "Decommissioned"
+            ],
+            "x-ms-enum": {
+                "name": "AccessibilityState",
+                "modelAsString": true,
+                "values": [
+                    {
+                        "description": "The offer or plan is private. Only an Admin can see and manage it.",
+                        "name": "Private",
+                        "value": "Private"
+                    },
+                    {
+                        "description": "The offer or plan is public. Users can see and self-subscribe to it.",
+                        "name": "Public",
+                        "value": "Public"
+                    },
+                    {
+                        "description": "The offer or plan is decommissioned. Current subscriptions can still consume it but new subscriptions cannot.",
+                        "name": "Decommissioned",
+                        "value": "Decommissioned"
+                    }
+                ]
+            }
+        },        
+        "DelegatedProviderOfferList": {
+            "description": "List of delegated provider offers.",
+            "properties": {
+                "value": {
+                    "description": "Array of delegated provider offers.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/DelegatedProviderOffer"
+                    }
+                },
+                "nextLink": {
+                    "type": "string",
+                    "description": "Continuation token"
+                }
+            }
+        }
+    },
+    "parameters": {
+        "DelegatedProviderOfferParameter": {
```

Comment:
I don't see this being referenced anywhere.

Reply:
removed